apps/hash-frontend/src/pages/shared/entity-type-page/type-slide-over-stack/type-slide-over-slide.tsx

```diff
@@ -28,6 +30,70 @@ import { EntityTypeHeader } from "../shared/entity-type-header";
 import { getTypesWithoutMetadata } from "../shared/get-types-without-metadata";
 import { useEntityTypeValue } from "../use-entity-type-value";
 
+const CopyableOntologyChip: FunctionComponent<{
+  entityType: EntityTypeWithMetadata;
+}> = ({ entityType }) => {
+  const [copyTooltipIsOpen, setCopyTooltipIsOpen] = useState(false);
+  const [copiedEntityTypeUrl, setCopiedEntityTypeUrl] = useState(false);
+
+  const ontology = parseUrlForOntologyChip(entityType.schema.$id);
+
+  const handleCopyEntityTypeUrl = useCallback(() => {
+    void navigator.clipboard.writeText(entityType.schema.$id);
```

Comment:
Technically this might throw an error if clipboard access is disabled... we could make `copyTooltipIsOpen` into `copyTooltip` to hold a string, and if the promise fails set it to 'Your security settings prevent clipboard access' or something like that.

Reply:
`error-fallback.tsx` has something similar

Reply:
Good point, I've accounted for this in https://github.com/hashintel/hash/pull/3860/commits/0c9795f45ebece56a7c1f410025f950f5af2f7d1